src/WebClient.spec.js

```diff
@@ -390,7 +400,7 @@ describe('WebClient', function () {
           // verify that any requests after maxRequestConcurrency were delayed by the responseDelay
           const queuedResponses = responses.slice(3);
           const minDiff = concurrentResponses[concurrentResponses.length - 1].diff + responseDelay;
-          queuedResponses.forEach(r => assert.isAbove(r.diff, minDiff));
+          queuedResponses.forEach(r => assert.isAtLeast(r.diff, minDiff));
```

Comment:
was this actually causing a failure? i think your change is more technically correct, but i'm wondering if you actually got a test run where the timers were so perfect that `r.diff` and `minDiff` were actually equal.

Reply:
I can confirm that I've run into the issue of getting a test run where the timers matched up and a false positive was thrown--just once, but it has happened.

Reply:
yes, I added this fix to the PR, because the build of my first commit broke accidentally